modules/charon-core/src/main/java/org/wso2/charon3/core/objects/plainobjects/MultiValuedComplexType.java

```diff
@@ -0,0 +1,164 @@
+package org.wso2.charon3.core.objects.plainobjects;
+
+/**
+ * author Pascal Knueppel <br>
+ * created at: 19.09.2018 - 09:51 <br>
+ * <br>
+ * this class representation can be used to easily add a multi valued complex type representation like an email
+ * or a phonenumber to an {@link org.wso2.charon3.core.objects.AbstractSCIMObject} object
+ */
+public class MultiValuedComplexType {
+
+    /**
+     * A label indicating the attribute's function, e.g., "work" or "home".
+     */
+    private String type;
+
+    /**
+     * A Boolean value indicating the 'primary' or preferred attribute value for this attribute, e.g., the
+     * preferred mailing address or the primary email address. The primary attribute value "true" MUST appear
+     * no more than once. If not specified, the value of "primary" SHALL be assumed to be "false".
+     */
+    private boolean primary;
+
+    /**
+     * A human-readable name, primarily used for display purposes and having a mutability of "immutable".
+     */
+    private String display;
+
+    /**
+     * The attribute's significant value, e.g., email address, phone number.
+     */
+    private String value;
+
+    /**
+     * The reference URI of a target resource, if the attribute is a reference. URIs are canonicalized per
+     * Section 6.2 of [RFC3986]. While the representation of a resource may vary in different SCIM protocol
+     * API versions (see Section 3.13 of [RFC7644]), URIs for SCIM resources with an API version SHALL be
+     * considered comparable to URIs without a version or with a different version. For example,
+     * "https://example.com/Users/12345" is equivalent to "https://example.com/v2/Users/12345".
+     */
+    private String reference;
+
+    public MultiValuedComplexType() {
```

Comment:
We could remove the constructor as it is default.

Reply:
this might become a problem because I defined an empty constructor and a constructor containing all parameters